README.md

```diff
@@ -1,3 +1,33 @@
 # dex-operator
 
+This is a operator-sdk based operator to deploy Dex. The development is focused on OpenShift, but hopefully, this can be a generic operator that runs just as fine on Kubernetes.
+
+# Usage
+
+You'll need operator-sdk 1.9.0+.
```

Comment:
Nit: I'd add also the go version (one can see it in go.mod I know) but next time you modify all this...